index.js

```diff
@@ -105,3 +106,27 @@ exports.stake = async function(options) {
     const result = await account.stake(options.publicKey, BigInt(options.amount));
     console.log('Result: ', JSON.stringify(result));
 }
+
+exports.login = async function(options) {
+    if (!options.walletUrl) {
+        console.log("Log in is not needed on this environment. Please use appropriate master account for shell operations.")
```

Comment:
this is hard to understand for outsider. Probably makes sense to link to docs on how to run local setup.